gui-doc/scribblings/framework/color.scrbl

```diff
@@ -270,6 +270,14 @@
     editor.
 
   }
+  @defmethod[(get-matching-paren-string [paren-str string?]
+                                        [get-side (or/c 'open 'close 'either) 'either])
+             (or/c string? #f)]{
+    Returns a string of a paren matching the other side of @racket[paren-str], if one exists
+    on the side indicated by @racket[get-side], or @racket[#f] if there is none.
+    If @racket[paren-str] includes any characters other than a single paren token (including
+    whitespace), this method returns @racket[#f].
+  }
```

Comment:
Probably good to connect this to the `pairs` argument of `start-colorer`?

Reply:
Thanks, will do!